discovery-provider/plugins/notifications/src/processNotifications/mappers/create.ts

```diff
@@ -235,6 +256,9 @@ export class Create extends BaseNotification<CreateNotificationRow> {
     let user
     if (this.trackId) {
       const track = resources.tracks[this.trackId]
+      if ('usdc_purchase' in (track.premium_conditions || {})) {
```

Comment:
Why is this logic different than the logic in `processNotification`? Shouldn't it also have the feature flag check? Or are we skipping email notifs for these forever?

Reply:
this is just "daily" & "weekly" emails. it's harder to queue there, and very few users have that setting

Reply:
We'll probably still want to send track create email summary notifs for premium tracks once the feature flag is on? Or are we planning to remove this check before going live?

Reply:
(default being `live` emails)

Reply:
Even if we don't queue these, we probably don't want to skip including Premium tracks from these forever, right?

Reply:
yeah i'll just revert and redeploy !